easybuild/easyconfigs/g/GAMESS-US/rungms.patch

```diff
@@ -0,0 +1,145 @@
+This patch adds support for the SLURM scheduler (without removing support for
+the PBS or SGE schedulers), while also setting other important values (such as
+scratch directories) and patching the rungms script so that other hardcoded
+values can be set by means of a regular expression during the EasyBuild
+configuration process.
+You may need to tweak the values for SCR and USERSCR to adjust them for your
+own site.
+
+Written by Benjamin Roberts, New Zealand eScience Infrastructure
+University of Auckland, Auckland, New Zealand
+--- gamess.orig/rungms	2013-05-18 04:15:11.000000000 +1200
++++ gamess/rungms	2015-01-29 17:39:53.359679532 +1300
+@@ -59,17 +59,23 @@
+ #       both Sun Grid Engine (SGE), and Portable Batch System (PBS).
+ #       See also a very old LoadLeveler "ll-gms" for some IBM systems.
+ #
+-set TARGET=sockets
+-set SCR=/scr/$USER
+-set USERSCR=~$USER/scr
+-set GMSPATH=/u1/mike/gamess
++set TARGET=self.connectivity
```

Comment:
I think there is, yes, see https://github.com/hpcugent/easybuild-easyblocks/pull/558/files#diff-cd8b197deed68d12e1fc6fdc3bdc5d1dR171.

This patch should only contain the SLURM bits, everything else should be handled by the easyblock itself.

In my enhanced version of this easyblock in https://github.com/hpcugent/easybuild-easyblocks/pull/558, I added a custom `scratch_dir` easyconfig parameter that people can tweak from their easyconfig file. That's a lot easier than having to change something in a patch file, imho.